test/stancjs/stancjs.expected

```diff
@@ -53,6 +53,448 @@ Identifier 'z' not in scope.
 Semantic error in 'string', line 6, column 4 to column 5: 
 Identifier 'z' not in scope.
 
+$ node functions-only.js
```

Comment:
I tried to avoid adding generated C++ in the Javascript tests because that essentially makes `node` a dependency for all stanc3 developers and we should then just make JS tests run on dune runtest.

I am guessing just the fact that this doesn't error out is enough? 

If you think we can't avoid it, then I guess we roll with it.

Reply:
Yeah it would be enough. I'll remove it